server/package.json

```diff
@@ -11,15 +11,15 @@
   "dependencies": {
     "connect-session-sequelize": "^7.0.4",
     "cookie-parser": "^1.4.5",
-    "dotenv": "^8.2.0",
+    "dotenv": "^8.6.0",
```

Comment:
Changing dependencies could be the source of difficult to trace bugs if happen to work in a team environment.